lib/consignment.js

```diff
@@ -251,7 +252,7 @@ var fetchConsignment  = function(args, connectionInfo, retryCounter) {
     log.debug('retry # ' + retryCounter);
   }
 
-  var path = '/api/1.0/consignment/' + args.apiId.value;
+  var path = '/api/2.0/consignments/' + args.apiId.value;
```

Comment:
have you considered creating a new path instead of replacing the existing ones?

Reply:
No